sdk/python/kfp/compiler/compiler_test.py

```diff
@@ -419,7 +419,7 @@ def test_set_description_through_pipeline_decorator(self):
 
         @dsl.pipeline(description='Prefer me.')
         def my_pipeline():
-            """Don't prefer me"""
+            """Don't prefer me."""
```

Comment:
Could you please revert any changes unrelated to the PR's main changes?

Reply:
Thanks for taking a look. Done - I've reverted all changes to compiler_test.py, and some of the yapf/docformatter changes made to artifact_types.py